compiler/types.nim

```diff
@@ -624,6 +628,21 @@ proc firstOrd*(conf: ConfigRef; t: PType): BiggestInt =
     internalError(conf, "invalid kind for firstOrd(" & $t.kind & ')')
     result = 0
 
+
+proc firstFloat*(t: PType): BiggestFloat =
+  case t.kind
+  of tyFloat..tyFloat128: -Inf
+  of tyRange:
+    assert(t.n != nil)        # range directly given:
+    assert(t.n.kind == nkRange)
+    getFloatValue(t.n.sons[0])
+  of tyVar: firstFloat(t.sons[0])
+  of tyGenericInst, tyDistinct, tyTypeDesc, tyAlias, tyStatic, tyInferred:
```

Comment:
At least ``tySink``  is missing in this list.

Reply:
updated